src/order/order.service.ts

```diff
@@ -38,13 +39,12 @@ export class PaymentService {
           type: phone.type,
           country: phone.country,
           area: phone.area,
-          number: phone.number.padStart(9, '0'), // Ensure the number is at least 8 digits long
+          number: phone.number.padStart(9, '0'),
         })),
       };
-  
-      // Log para visualizar os dados de phone após formatação
+
       this.logger.debug('Dados de phone após formatação:', JSON.stringify(customer.phones));
```

Comment:
Follow a pattern, most of the logs are written in english like the Programming Language native lang. Always follow a pattern for better coding. Update it to english